client/my-sites/plugins/plugin-site-list/index.jsx

```diff
@@ -15,50 +13,52 @@ import isConnectedSecondaryNetworkSite from 'calypso/state/selectors/is-connecte
 
 import './style.scss';
 
-export class PluginSiteList extends Component {
-	static propTypes = {
-		plugin: PropTypes.object,
-		sites: PropTypes.array,
-		sitesWithSecondarySites: PropTypes.array,
-		title: PropTypes.string,
-	};
```

Comment:
Let's add the propTypes for the functional component too, like here 
https://github.com/Automattic/wp-calypso/blob/f2e6f88ae65c8571bbac21b284c29b48ec987598/apps/notifications/src/panel/templates/action-button.jsx#L22-L29